core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMaterializedViewDefinition.java

```diff
@@ -216,13 +223,20 @@ public boolean equals(Object o)
             }
             Column column = (Column) o;
             return Objects.equals(name, column.name) &&
-                    Objects.equals(type, column.type);
+                    Objects.equals(type, column.type) &&
+                    Objects.equals(comment, column.comment);
         }
 
         @Override
         public int hashCode()
         {
-            return Objects.hash(name, type);
+            return Objects.hash(name, type, comment);
+        }
+
+        @Override
+        public String toString()
+        {
+            return name + " " + type;
         }
```

Comment:
to be honest not sure this change make sense without all other logic. For sure it will not break the compilation. But I d prefer to keep it inside the commit with other changes.